orttraining/orttraining/core/session/training_session.h

```diff
@@ -182,6 +182,8 @@ class TrainingSession : public InferenceSession {
       // cut_list contains the list of CutInfo to make the graph partitions.
       // cut_list[i] contains the CutInfo to make the partition between stage i and stage i+1
       std::vector<CutInfo> cut_list;
+      // Alternative for partition. We map operators to stage ids. 
+      std::map<std::string, int> op_id_to_stage;
```

Comment:
We need to add a comment to explain the format of a non-trivial field. For example, op_id_to_stage["MyMatMul"] is the pipeline stage a node called "MyMatMul" belongs to.